Makefile

```diff
@@ -146,12 +146,24 @@ TEST_APK_PROJECTS = \
 	tests/CodeGen-Binding/Xamarin.Android.JcwGen-Tests/Xamarin.Android.JcwGen-Tests.csproj \
 	tests/locales/Xamarin.Android.Locale-Tests/Xamarin.Android.Locale-Tests.csproj
 
-# Syntax: $(call BUILD_TEST_APK,path/to/project.csproj)
+TEST_APK_PROJECTS_RELEASE = \
+	src/Mono.Android/Test/Mono.Android-Tests.csproj
+
+# Syntax: $(call BUILD_TEST_APK,path/to/project.csproj,additional_msbuild_flags)
 define BUILD_TEST_APK
 	# Must use xabuild to ensure correct assemblies are resolved
-	MSBUILD="$(MSBUILD)" tools/scripts/xabuild $(MSBUILD_FLAGS) /t:SignAndroidPackage $(1)
+	MSBUILD="$(MSBUILD)" tools/scripts/xabuild $(MSBUILD_FLAGS) /t:SignAndroidPackage $(2) $(1)
 endef	# BUILD_TEST_APK
 
+# Syntax: $(call RUN_APK_TESTS,projects,additional_msbuild_flags)
+define RUN_APK_TESTS
+	$(foreach p, $(1), $(call BUILD_TEST_APK, $(p),$(2)))
+	$(MSBUILD) $(MSBUILD_FLAGS) $(2) tests/RunApkTests.targets
+endef
+
 run-apk-tests:
-	$(foreach p, $(TEST_APK_PROJECTS), $(call BUILD_TEST_APK, $(p)))
-	$(MSBUILD) $(MSBUILD_FLAGS) tests/RunApkTests.targets
+	$(call RUN_APK_TESTS, $(TEST_APK_PROJECTS), )
+ifneq ($(wildcard bin/Release),)
```

Comment:
This shouldn't be necessary, unless the intent is to prevent these tests from running on the PR builder.

The `bin/Debug/...` artifacts should be able to build an app with `/p:Configuration=Release`.